src/dmd/mtype.d

```diff
@@ -2560,7 +2560,12 @@ extern (C++) abstract class Type : RootObject
                 if (s)
                     error(loc, "no property `%s` for type `%s`, did you mean `%s`?", ident.toChars(), toChars(), s.toPrettyChars());
                 else
-                    error(loc, "no property `%s` for type `%s`", ident.toChars(), toChars());
+                {
+                    if (ident == Id.call && ty == Tclass)
+                        error(loc, "no property `%s` for type `%s`, did you mean `new %s`?", ident.toChars(), toChars(), toPrettyChars());
```

Comment:
BTW any reason why `toChars` is always fully qualified for classes, but not for structs?

Reply:
I don't know.  I suppose that should also be addressed (in another PR).